english/call-a-doctor/doctor.md

```diff
@@ -5,7 +5,7 @@ color, not like the institutionalized white you remember from movies.
 
 [The doctor](https://www.youtube.com/watch?v=WsGDvMqcP94) enters.
 
-["Let me outta here, Doc!!!"](let-me-out/let-me-out.md)
```

Comment:
It looks like there was already a `let-me-out.md`: https://github.com/udacity/create-your-own-adventure/blob/master/english/call-a-doctor/let-me-out/let-me-out.md. Maybe you could add to that file instead of creating a new one?